Library/Homebrew/cmd/audit.rb

```diff
@@ -23,65 +26,51 @@
 require "date"
 
 module Homebrew
+  RUBY_2_OR_LATER = RUBY_VERSION.split(".").first.to_i >= 2
```

Comment:
This seems to be used multiple times thoroughout the codebase. We probably need a method in `utils.rb` for this.